### PR TITLE
Remove unnecessary read, remove output from git

### PR DESCRIPTION
**Pull request by mridulm**

### CODE REVIEW

File: src/main/scala/SparkEvents.scala [hunk 1]
```diff
@@ -70,15 +70,11 @@ object SparkEvents {
     }
   }
 
-  def writeEventsToFile(
-      inputStream: InputStream,
-      outputPath: Path,
-      hadoopConf: Configuration): Unit = {
+  def writeEventsToFile(inputStream: InputStream, outputPath: Path, hadoopConf: Configuration): Unit = {
```
Comment (mridulm) [inline src/main/scala/SparkEvents.scala:73]: The format change due to scalafmt rules in project

File: src/main/scala/SparkEvents.scala [hunk 2]
```diff
@@ -70,15 +70,11 @@ object SparkEvents {
...
     val fs = outputPath.getFileSystem(hadoopConf)
     val fos = fs.create(outputPath)
     try {
       val buf = new Array[Byte](8192)
-      var len = inputStream.read(buf)
```
Comment (mridulm) [inline src/main/scala/SparkEvents.scala:81]: This is the single line change I wanted to make - ended up being a massive PR :-)

Comment (venkata91) [reply src/main/scala/SparkEvents.scala:81]: Just seeing this PR :) Sorry for the delayeddddd response :-)

**Review by venkata91 — APPROVED** [review]
LGTM

**Pull request merged**